src/main/java/duke/Event.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Event class.
+ * Represents a event task.
+ * Extends from Task.
+ * Contains event name, date, start time and end time.
+ *
+ * @author YanCheng
+ */
+public class Event extends Task {
+
+    private LocalDate date;
+    private LocalTime startTime;
+    private LocalTime endTime;
+
+    /**
+     * Constructor for Event
+     * @param taskName Name of Event
+     * @param date Date of Event in format of "YYYY-MM-DD"
+     * @throws DukeException If date is not in the specified format
+     */
+    public Event(String taskName, String date) throws DukeException {
+        // date format must be in YYYY-MM-DD e.g. 2020-08-22
+        // time format must by in XX:XX-YY:YY e.g. 14:00-16:00
+        super(taskName.stripTrailing());
+        String[] arr = date.split(" ");
+        String eventDate = arr[0];
+        String[] eventTime = arr[1].split("-");
+        String eventStartTime = eventTime[0];
+        String eventEndTime = eventTime[1];
+        try {
+            this.date = LocalDate.parse(eventDate);
+            this.startTime = LocalTime.parse(eventStartTime);
+            this.endTime = LocalTime.parse(eventEndTime);
+        } catch (DateTimeParseException e) {
+            throw new DukeException("Date and time is in the wrong format");
+        }
+    }
+
+    /**
+     * Return deadline information to be stored locally.

```

Comment:
I think it complies with the coding standards better if you write 'Returns' (third person singular tense of the word for header) instead of 'Return'? I noticed the same issue in several other places of documentation too.